.profile

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+host=$(echo "$VCAP_SERVICES" | jq -r '.["aws-rds"][] | select(.name == "keycloak-db") | .credentials.host')
+db_name=$(echo "$VCAP_SERVICES" | jq -r '.["aws-rds"][] | select(.name == "keycloak-db") | .credentials.db_name')
+username=$(echo "$VCAP_SERVICES" | jq -r '.["aws-rds"][] | select(.name == "keycloak-db") | .credentials.username')
+password=$(echo "$VCAP_SERVICES" | jq -r '.["aws-rds"][] | select(.name == "keycloak-db") | .credentials.password')
+jdk_version=$(cat JDK_VERSION)
+
+export KC_DB_URL_HOST=$host
+export KC_DB_URL_DATABASE=$db_name
+export KC_DB_USERNAME=$username
+export KC_DB_PASSWORD=$password
+export PATH=$PATH:/home/vcap/app/jdk-$jdk_version/bin
+
+export KC_HTTPS_CERTIFICATE_FILE=$CF_INSTANCE_CERT
+export KC_HTTPS_CERTIFICATE_KEY_FILE=$CF_INSTANCE_KEY
```

Comment:
We want Keycloak performing it's own TLS instead of using the Envoy proxy?

Reply:
Keycloak is listening on both 8080 for http and 8443 for https. 8080 is https via the proxy. When connecting via ssh to admin interface, 8443 can be used